src/GitUtilities.js

```diff
@@ -20,8 +20,7 @@ export default class GitUtilities {
 
   @logger.logifySync()
   static commit(message) {
-    // Use echo to allow multi\nline strings.
-    ChildProcessUtilities.execSync("git commit -m \"$(echo \"" + message + "\")\"");
+    ChildProcessUtilities.execSync("git commit -m \"" + message + "\"");
```

Comment:
We had more success with this change: https://github.com/florianorpeliere/lerna/commit/86d53258ac084deb954daffca8a0081a593d7d0f